### DISCUSSION TITLE
Possibility to wrap generated constructor params

### DISCUSSION THREAD

**Discussion by Hantick** [riok/mapperly#1420] [Q&A]
Is there a way to setup a config to wrap every parameter in the new line of the generated code?
I have my implementation based on `record` type, that would be helpful.

So instead:
```
var target = new Model(param1, param2, param3, param4, ... param40);
```
it will be:
```
var target = new Model(
param1,
param2,
param3,
param4,
...
param40
```

or even:
```
var target = new Model(
Param1: param1,
Param2: param2,
DifferentParam3: param3,
Param4: param4,
...
Param40: param40
```

That would be a lot easier to read and understand

**Comment by latonz** [comment]
This seems to be a duplicate of https://github.com/riok/mapperly/issues/1322 which is worked on in https://github.com/riok/mapperly/pull/1337.
